### DISCUSSION TITLE
--upgrade deleted custom notification mp3 in html

### DISCUSSION THREAD

**Discussion by brknsoul** [vladmandic/sdnext#1088] [General]
Upon a suggestion, I placed a custom notification mp3 file in `html` called `customnotification.mp3` but it got deleted when I did an `--upgrade`.

**Comment by tkiikt** [comment]
Settings -> Live previews -> Path to notification sound allows you to put sound files under any file name and in any location you like. I was also initially stuck with the stereotype of putting notification.mp3 in /html. "notification.mp3" is reserved by the system, so it is better to use the original .mp3 and location path.

**Reply by brknsoul** [reply]
>  "notification.mp3" is reserved by the system

I know this, that's why I called it `customnotification.mp3`, so it wouldn't get overwritten, but instead it got deleted.
I imagine I could call it `flurp-nurp.mp3` and it'd still get deleted.

In the mean time, I just moved it outside the `automatic` folder and pointed the setting to it.

**Reply by tkiikt** [reply]
I had previously requested to have .mp3 added to the ignore list, but maybe it's a bug.

**Reply by brknsoul** [reply]
I had tried editing the .gitignore file, but it seems that gets overwritten too!

**Reply by vladmandic** [reply]
you cannot place custom files just anywhere in the repo - anything placed in `/html` WILL get removed.
`.gitignore` already includes specs to ignore any mp3 files you place in ROOT of the app.

**Reply by brknsoul** [reply]
I guess. I imagine that's why the original code looked for notification.mp3 in the root directory, so a user could place one there and not have it blasted by updates.

Perhaps update .gitignore to ignore mp3s in `/html`, since that's where the main one resides?

**Reply by vladmandic** [reply]
i don't want to ignore any files inside core folders. root already has plenty of exclusions to allow for custom batch or shell scripts or mp3 files, etc...
but this would be a good FAQ entry.

**Reply by brknsoul** [reply]
Okay. Also, could you add ignoring `*.lnk` files in the root folder?
Otherwise Windows link files (created by my entry in the FAQ) won't persist through updates.

**Reply by vladmandic** [reply]
yes, i can do that. anything else? i'm totally ok increasing ignore list for root folder.

**Reply by brknsoul** [reply]
I think that's it. If there's anything I can think of, I'll reply here. Thanks.
